### ISSUE TITLE
Keyboard Shortcut

### ISSUE THREAD

**Issue by poxin13** [amiantos/pibar#35]
Is there a way to customize the global keyboard shortcut?

**Comment by amiantos** [comment]
No, not at the moment. But you can set up a script and make a custom keyboard shortcut I believe.

https://github.com/amiantos/pibar/issues/5

**Comment by amiantos** [comment]
Perplexity has a nice box you can record or clear shortcuts into, would be nice if I could bring that to PiBar.
